husky_components/tabs/main.js

```diff
@@ -50,6 +50,7 @@ define(function() {
             forceReload: false,
             callback: null,
             forceSelect: true,
+            conditionData: {},
```

Comment:
And maybe we should not forget that we could call it just `data` now, which would still be better than `conditionData` IMO.